versions/3.0.4.md

```diff
@@ -1072,20 +1072,20 @@ These fields MAY be used with either `content` or `schema`.
 
 This object MAY be extended with [Specification Extensions](#specification-extensions).
 
-Note that while `"Cookie"` as a `name` is not forbidden with `in: header`, the effect of defining a cookie parameter that way is undefined; use `in: cookie` instead.
+Note that while `"Cookie"` as a `name` is not forbidden if `in` is `"header"`, the effect of defining a cookie parameter that way is undefined; use `in: "cookie"` instead.
 
 ###### Fixed Fields for use with `schema`
 
 For simpler scenarios, a [`schema`](#parameter-schema) and [`style`](#parameter-style) can describe the structure and syntax of the parameter.
-When `example` or `examples` are provided in conjunction with the `schema` object, the example SHOULD match the specified schema and follow the prescribed serialization strategy for the parameter.
+When `example` or `examples` are provided in conjunction with the `schema` field, the example SHOULD match the specified schema and follow the prescribed serialization strategy for the parameter.
 The `example` and `examples` fields are mutually exclusive, and if either is present it SHALL _override_ any `example` in the schema.
 
-Serializing with `schema` is NOT RECOMMENDED for `in: cookie` parameters, `in: header` parameters that use HTTP header parameters (name=value pairs following a `;`) in their values, or `in: header` parameters where values might have non-URL-safe characters; see [Appendix D](#appendix-d-serializing-headers-and-cookies) for details.
+Serializing with `schema` is NOT RECOMMENDED for `in: "cookie"` parameters, `in: "header"` parameters that use HTTP header parameters (name=value pairs following a `;`) in their values, or `in: "header"` parameters where values might have non-URL-safe characters; see [Appendix D](#appendix-d-serializing-headers-and-cookies) for details.
```

Comment:
I'm curious about the convention we are using for string values here and elsewhere. The original text has `in: cookie", which in YAML is valid and I think is preferred over the new text `in: "cookie"`. The new text is not valid JSON. So what is our convention and why did we choose this?

Reply:
@mikekistler Please see the PR description:

> 5. fixed fields are monospaced

This is the predominant style for fixed fields in the existing text.

> 6. field values are monospaced in JSON notation: `true`, `false`, `null`, `"header"`, ...

This is done for example in the first table in section https://spec.openapis.org/oas/v3.0.3.html#fixed-fields-9, and I find it helpful that field `names` and field `"values"` are clearly distinguished, so I applied this pattern wherever I wondered whether something is a field name or a field value.

The second table in the same section does not use that convention and is rather hard to read. For example

> When `style` is `form`, the default value is `true`.

Here `form` is the content of a JSON string: is `true` also the content of a JSON string, or is it a JSON boolean? How much context do I have to read to figure that out?

The pattern `in: query` was introduced by Henry in 3.0.4, and I first tried to rephrase that into a style already present in 3.0.3:
> if `in` is `"query"`

That felt clumsy in some places, so I opted for `in: "query"` etc., combining conventions 5 and 6.

The PR description is my stab at an "editor's style guide", and of course open for discussion, and destined for documentation, for example in `CONTRIBUTING.md`

Reply:
Sorry I glossed over the PR description, and thank you for writing this! 

On the particular style choice, I think I would slightly prefer YAML to JS, just to avoid adding another syntax style to the doc, but I can live with either as long as we are consistent.